### PR TITLE
Microverse survey form

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
This is a microverse project where I was required to create a single web page.
* Add README.md file

### CODE REVIEW

**Review by mimipeshy — CHANGES_REQUESTED** [review]
Hi @hamid-murambiwa 👋🏾 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you did not make a review at all, I can see only an open pull request. Kindly do a review of this project. You can find resources on [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by hamid-murambiwa** [comment]
The README.md file is empty. Please add descriptive content to the file.

**Review by hamid-murambiwa — COMMENTED** [review]
The README.md file is empty. Please add descriptive content to the file.

**Review by hamid-murambiwa — COMMENTED** [review]
Hi @hamid-murambiwa

To Highlight

The commit messages are clear.
The PR is descriptive.
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Required Changes ♻️

Check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @hamid-murambiwa , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] Kindly have atleast 2 comments in the review.
- [ ] Kindly make use of the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments.
     * Make sure that the comment is under a line or block of code that fits the context the comment is about.

- [ ] Your comments should be phrased politely, as suggestions instead of orders. For example:

* This is a polite suggestion: *"I think it would be better to use semantic tags"*
* This reads as an order: *"Use semantic tags"*

- [ ] You should always include an explanation, a "why", for each requested change.  It might be obvious for you but not for someone else.
- [ ] Your comments are clear and easy to understand for example
* This is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
* This is a vague comment: *The README.md file is empty. Please add descriptive content to the file.*
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comments under one single summary  😉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1>survey-page</h1>
+   *Portfolio main page with a mobile and main page*
```
Comment (hamid-murambiwa) [inline README.md:2]: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

File: style.css [hunk 2]
```diff
@@ -22,5 +22,3 @@ html {
     margin-top: 5%;
     background-color: rgba(138, 46, 224, 0.5);
 }
-
-
```
Comment (hamid-murambiwa) [inline style.css:26]: Please make sure you fix all the linters errors

**Review by hamid-murambiwa — COMMENTED** [review]
Hi @hamid-murambiwa

To Highlight

The commit messages are clear.
The PR is descriptive.
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Required Changes ♻️

Check the comments under the review.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @hamid-murambiwa  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**